e2e/provision/distros_supported.yml

```diff
@@ -21,3 +21,7 @@ fedora_34:
   reload: true
   project_id: fedora-cloud
   family: fedora-cloud-34
+fedora_41:
```

Comment:
SHould we remove fedora_24 above?

Reply:
Good point. removed it.

Reply:
Just make sure that fedora41 works with free5gc given its kernel constraints